spec/controllers/ops_controller/settings/schedules_spec.rb

```diff
@@ -30,7 +30,7 @@
       let(:filter_type) { "ems" }
 
       before do
-        allow(ExtManagementSystem).to receive(:find).with(:all, {}).and_return([ext_management_system])
+        allow(ExtManagementSystem).to receive(:where).with({}).and_return([ext_management_system])
```

Comment:
Did these tests fail without any code change?  I'm afraid mocking the actual code is causing us to not run this code on upgrades.